server/routes/router.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const router = express.Router();
+const loginRouter = require('./auth/auth');
+const adsRouter = require('./api/ads');
+
+router.use("/", loginRouter);
+router.use("/ads" , adsRouter);
```

Comment:
Remove 1 space near ` , `

Also, Do you mean adds (As in addition + )? Because an ad is like an ad blocker, add advertisements.

Reply:
ad stands for advertisement,
IMO it's advertisements are too long to write every time you use it

Reply:
My bad... I worked too much with adding a post I thought it is add 😅 

`/ads` as shortcut is great!!